Dockerfile

```diff
@@ -17,10 +17,10 @@ RUN apk add --no-cache tini
 # Tini is now available at /sbin/tini
 
 WORKDIR /app
-COPY --from=builder /dist/github-migration-cli .
+COPY --from=builder /dist/gitsink .
 
 # Run your program under Tini
 # Configure runtime behavior
 ENV PATH="/app:${PATH}"
 ENTRYPOINT ["/sbin/tini", "--"]
-CMD ["github-migration-cli"]
\ No newline at end of file
+CMD ["gitsink"]
```

Comment:
Should: always finish every file with a newline.  That's why Github is marking it.